build.gradle

```diff
@@ -1,26 +1,21 @@
 buildscript {
```

Comment:
Nit: Commit title is slightly misleading, when doing a bisect lookin for a commit that broke something this could easily be overlooked